lbz/events/broker.py

```diff
@@ -40,3 +44,55 @@ def _get_handlers(self) -> List[Callable]:
             return self.mapper[self.event.type]
         except KeyError as err:
             raise NotImplementedError(f"No handlers implemented for {self.event.type}") from err
+
+
+class EventBroker(BaseEventBroker):
+    def __init__(
+        self,
+        mapper: Mapping[str, List[Callable[[Event], None]]],
+        event: dict,
+        context: LambdaContext,
+        *,
+        type_key: str = EVENT_BRIDGE_DEFAULT_TYPE_KEY,
+        data_key: str = EVENT_BRIDGE_DEFAULT_DATA_KEY,
+    ) -> None:
+        super().__init__(
+            mapper,
+            event,
+            context,
+            type_key=type_key,
+            data_key=data_key,
+        )
+
+
+class EventBridgeBroker(BaseEventBroker):
```

Comment:
According to the above comment, this class will become redundant - we can remove it.